src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -68,5 +70,6 @@ public static class EnvironmentSettingNames
 
         public const string KubernetesServiceHost = "KUBERNETES_SERVICE_HOST";
         public const string KubernetesServiceHttpsPort = "KUBERNETES_SERVICE_PORT_HTTPS";
+        public const string MeshInitURI = "MESH_INIT_URI";
```

Comment:
This should be SCM_RUN_FROM_PACKAGE

Reply:
no, this is used for something different. I'm writing an explanation now